### DISCUSSION TITLE
How to change Series Name?

### DISCUSSION THREAD

**Discussion by sergiu-gheorghe** [ShapeCrawler/ShapeCrawler#250] [Q&A] [answered]
```
 var chart = (IChart)slide.Shapes.First(sp => sp is IChart && sp.Name == chartName);
 var series = chart.SeriesCollection[i];
 series.Name = "change"; // Cannot change the property as it doesn't have a setter
```

Is it possible somehow to change the Series Name?

**Comment by AdamShakhabov** [comment, marked as answer]
As you noticed, the updating of the series name is not supported yet.

Unfortunately, there is not some light workaround for this, because it leads to manipulating not only presentation. You should also update the underlying Spreadsheet.

**Reply by sergiu-gheorghe** [reply]
OK, thank you for the explanation.

**Reply by sergiu-gheorghe** [reply]
Does the library support loading ChartData from the excel Spreadsheet?

**Reply by AdamShakhabov** [reply]
You can read byte array of the underlining Spreadsheet via `IChart.WorkbookByteArray`, but this helps you only _read_ Spreadsheet data, not _update_.
